tests/config/locales.tests.ts

```diff
@@ -1,65 +1,46 @@
-import {readFile as fsReadFile, readdir as fsReadDir} from 'fs';
+import {readFileSync as fsReadFile, readdirSync as fsReadDir} from 'fs';
```

Comment:
Not sure if we want using a Sync due to.
```
You should never call readFileSync in a node.js since it will tie up the single thread loop while I/O is performed. You want the node loop to process other requests until the I/O completes and your callback handling code can run.
```

Reply:
Aw, that's a bummer. :/ describe() cannot be async, and it.each() can't have an array dynamically populated. So I can't use promises to define test structure at all.

Reply:
:/